test/functional/content_credentials/list_test.rb

```diff
@@ -0,0 +1,68 @@
+require File.join(File.dirname(__FILE__), '../test_helper')
+require File.join(File.dirname(__FILE__), '../lifecycle_environment/lifecycle_environment_helpers')
+
+require 'hammer_cli_katello/content_view_puppet_module'
+
+describe 'listing repositories' do
+  include LifecycleEnvironmentHelpers
+
+  before do
+    @cmd = %w(content-credentials list)
+  end
+
+  let(:org_id) { 1 }
+  let(:lifecycle_env_id) { 1 }
+  let(:empty_response) do
+    {
+      "total" => 0,
+      "subtotal" => 0,
+      "page" => "1",
+      "per_page" => "1000",
+      "error" => nil,
+      "search" => nil,
+      "sort" => {
+        "by" => nil,
+        "order" => nil
+      },
+      "results" => []
+    }
+  end
+
+  it "lists an organizations keys" do
+    params = ["--organization-id=#{org_id}"]
+
+    ex = api_expects(:content_credentials, :index, 'Organizations repositories list') do |par|
```

Comment:
This will be very confusing unless we update the expectation description

```suggestion
    ex = api_expects(:content_credentials, :index, 'Content credentials list') do |par|
```

Check for other instances of 'repository' in this test file and change to content credentials accordingly.